warn/scrapers/mi.py

```diff
@@ -80,6 +79,23 @@ def scrape(
             pdf_file = cache.download(cache_key, pdf_url)
         pdf_list.append(pdf_file)
 
+    # parse pdf data
+    pdf_data = []
+    print(pdf_list)
+    # Parse the pdfs from 2007-2015
+    for file in range(len(pdf_list) - 9, len(pdf_list))[::-1]:
```

Comment:
Can you talk me through what these index are doing?

Reply:
pdf_list is a list of file paths to be opened with pdf plumber. file is an index to access a file path in pdf_list to open using pdf plumber and process. Pdfs from 2000-2006 and 2007-2015 had to be processed differently due to formatting. I reversed the range so that the data can be in order on the csv.